README.md

```diff
@@ -14,4 +14,5 @@ Team *xos*:
 
 ## Development style guide
 
-- Use tabs for indentation as opposed to spaces
+- Minh Bui
```

Comment:
I think this was a mistake, right? :)

Reply:
the file probably automatically did that, sorry